js/data/currencies.js

```diff
@@ -651,6 +651,10 @@ function getIndexedCurrencies() {
   return _indexedCurrencies;
 }
 
+export function getListofCurrencies() {
+  return _.pluck(currencies, 'code');
+}
```

Comment:
This was added because checking a currency with getCurrencyByCode didn't work in the test when the currency was undefined. There may be a better way to do it, though it seems like a simple list of currencies is useful to have.